pbjam/asy_peakbag.py

```diff
@@ -377,12 +375,23 @@ def __call__(self, developer_mode):
         """
         self.developer_mode = developer_mode
         
-        self.fit = pb.mcmc(np.median(self.start_samples, axis=0), 
-                           self.likelihood, self.prior)
-        self.fit(start_samples=self.start_samples)
+        if method not in ['mcmc', 'nested']:
+            warnings.warn(f'Method {method} not found: Using method mcmc')
+            method = 'mcmc'
+
+        if method == 'mcmc':
+            self.fit = pb.mcmc(np.median(self.start_samples, axis=0),
+                               self.likelihood, self.prior)
+            self.fit(start_samples=self.start_samples)
+
+        elif method == 'nested':
+            bounds = [[self.start_samples[:, n].min(), 
```

Comment:
I think for now these bounds are necessary as the prior is not universal.  When we switch to a universal prior this will be better (I think)

Reply:
Probably fine for now.